routes/index.js

```diff
@@ -342,23 +338,38 @@ router.get('/search', function(req, res, next) {
             })
     }
 
-    function finalize(err, songs2d) {
+    function findInLibraryAndPlaylist(songs, done) {
+        User.findById(req.user._id, function(err, user) {
+            if (err) next(err)
+            Playlist.find({
```

Comment:
the name of the function is find in library and playlist, but I only see playlist here. What does that mean?